setup.py

```diff
@@ -12,12 +12,13 @@
 
 install_requirements = [
     "boto3>=1.3,<2.0",
-    "click==6.7",
+    "click>=7.0<8.0",
     "PyYaml==3.13",
     "Jinja2>=2.8,<3",
-    "packaging==16.8",
-    "colorama==0.3.7",
-    "six==1.11.0"
+    "packaging>=18.0<19.0",
+    "colorama>=0.3.7<1.0.0",
```

Comment:
Pre-stable packaging. Would definitely like to get this dependency to a stable release i.e. `1.0`. But for now a lot could have changed between `0.3.7` and `1.0` so I want to keep this pinned to `0.3.7` until we get v2 out and feedback from that has settled.

Reply:
Okay this is fair point. 
So could be updated to latest 0.4.0 ?

Reply:
Again, it is a pre-stable release so there are no guarantees about API compatibility between versions. Indeed looks like they have dropped Python 2.6 support in 0.4.0 which is something we still support. Happy for this to be pinned to `0.3.9` until we drop python 2 support.